docs/migration-guide-ec2.md

```diff
@@ -0,0 +1,206 @@
+
+    a. Ensure that the latest version of main has been deployed to CODE/PROD
+    before making further changes
+
+    b. Populate the AMI<App> CloudFormation parameter with the AMI ID that is
+    currently being used for your application
+
+    c. Preview the change set: it should only contain Add operations, as we are
+    just provisioning new resources here.
+
+5.  Deploy this change using Riff-Raff and confirm that both ASGs are updated
+    as part of the deployment.
+
+6.  Test your app’s functionality via the new load balancer’s DNS name (this can
+    be found as an output in the CFN console).
+
+## Stage 2: switch DNS
+
+_This stage ensures that you are able to manage your own DNS record(s) via CDK and updates your DNS record(s) to point at your new load balancer(s)._
+
+_At the end of this stage, all traffic should be running via the new resources that we created in Stage 2. The equivalent old resources should still be part of your stack, but they should be redundant by the end of this stage._
+
+_The instructions here assume that your DNS record is managed via NS1. To check if you are using NS1, you can use dig - dig +nssearch +noall +answer <TLD> where TLD is the top level domain. For example dig +nssearch +noall +answer gutools.co.uk._
+
+1.  Start managing your DNS record via CDK by instantiating a GuCname construct
+
+    Use dig to check the current properties of your DNS record, for example:
+
+    ```
+    $ dig +noall +answer CNAME security-hq.gutools.co.uk
+    $ security-hq.gutools.co.uk. 3600	IN	CNAME	secur-loadb-re7mdt13o53h-879373669.eu-west-1.elb.amazonaws.com.
+    ```
+
+    (3600 is the TTL, and `secur0loadb...` is the loadbalancer.)
+
+    Instantiate the GuCname class, ensuring that all properties match the
+    properties in NS1. The CNAME should still point at your old load balancer’s
+    DNS name for now.
+
+    There's an example of this
+    [here](https://github.com/guardian/security-hq/pull/336/files#diff-f38efbe4db4fb3d00ad6e3d6792a8d0e4fa818b3b974cb9d012ce2e81a242bfc).
+
+    _N.B. if your template already includes a Guardian::DNS::RecordSet resource,
+    you will need to:_
+
+    - _Use the same [logical
+      id](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html)
+      when instantiating the GuCname via CloudFormation_
+    - _Delete the Guardian::DNS::RecordSet resource in the same PR that the
+      GuCname is added_
+
+2.  Drop the TTL for your DNS record, to enable faster rollback if something goes wrong.
+
+    Wait for the TTL to expire before proceeding. ([Example
+    PR](https://github.com/guardian/security-hq/pull/338/files)).
+
+3.  Update DNS entry for your so that it points to the new load balancer
+    ([Example PR](https://github.com/guardian/security-hq/pull/339/files)).
+
+4.  Run some tests to confirm that functionality still works as expected.
+
+5.  Wait for a while before proceeding with Stage 6.
```

Comment:
Minor, this confused me at first because I was looking for a `## Stage 6: xyz` section

```suggestion
5.  Wait for a while before proceeding with the next step.
```